meta-mender-core/classes/mender-artifactimg.bbclass

```diff
@@ -55,6 +80,36 @@ IMAGE_CMD_mender () {
         image_flag=-f
     fi
 
+    if [ -n "${MENDER_ARTIFACT_NAME_DEPENDS}" ]; then
```

Comment:
I know we are basically repeating the same thing here, but I like the expliciity of it. Happy to turn it into a function though, if sought for.

Reply:
No, I also like it. It's still pretty minimal, but verbose enough to be explicit.